swagger_server/controllers/operand_controller.py

```diff
@@ -1,15 +1,27 @@
+import flask
+
 from swagger_server import util
+from swagger_server.models.evaluation import Operand
+from swagger_server.util import ValidationError
 
 
-def add_operand():  # noqa: E501
+def add_operand(payload):  # noqa: E501
     """Create an operand
 
     Creates an operand # noqa: E501
 
 
     :rtype: None
     """
-    return 'do some magic!'
+    if not payload:
+        raise ValidationError(400, 'Couldn\'t parse JSON POST body.')
+    if not payload.get('id'):
+        raise ValidationError(400, 'No id field specified')
+    try:
+        operator = Operand.create(expression_id=payload.expression_id, rank=payload.rank, value=payload.value, type=payload.type)
```

Comment:
Is payload an object here that has these properties (like `expression_id` and `rank`) or is it a dictionary that we have to index?